tests/Yaapii.Atoms.Tests/Scalar/TernaryTest.cs

```diff
@@ -71,5 +71,52 @@ public void ConditionFunc()
                     input => input = 2
                 ).Value() == 8);
         }
+
+        //
+
+        [Fact]
+        public void ConditionTrueOneGeneric()
+        {
+            Assert.True(
+                new Ternary<int>(
+                    new True(),
+                    6,
+                    16
+                ).Value() == 6);
```

Comment:
💅🏻 line breaks missing in every test in this line.
`);` in next line

Reply:
Thx!